Makefile

```diff
@@ -20,3 +20,27 @@ test:
 	@echo hello;
 
 .PHONY: clean $(TARGETS)
+
+
+PREFIX = /usr/local
+
+alib = libvmaf.a
```

Comment:
Can you move the logic here and below into wrapper/Makefile, and only make calls from this top-level Makefile?